src/NexusMods.App.UI/Resources/Language.pl.resx

```diff
@@ -456,6 +456,18 @@
 		<value>DODANE</value>
 	</data>
 	<data name="FileOriginsPageView_ActionHeader" xml:space="preserve">
-		<value>AKCJE</value>
-	</data>
+		<value>DZIAŁANIA</value>
+	</data>
+    <data name="DownloadStatusDesignViewModel_FormatStatus_Cancelled" xml:space="preserve">
+        <value>Anulowane</value>
+    </data>
+    <data name="DownloadInProgressView__Clear" xml:space="preserve">
+        <value>Wyczyść</value>
+    </data>
+    <data name="DownloadInProgressView__Clear_completed" xml:space="preserve">
+        <value>Wyczyść ukończone</value>
```

Comment:
It this how software usually phrases it? The translation is quite literal.

I was wondering if `wyczyszone` might not be more correct here.

Reply:
Clear - Wyczyść
Cleared - Wyczyszczone

I haven't checked the UI yet, but I suspect it is a button/option to clear completed downloads. Something similar to the Clear All option in the download history in the Chrome browser. 

Wyczyść is the future tense.
Wyczyszcozne is the past tense.

Reply:
I was thinking of `Clear Completed` as 'we just cleared this' hence using past tense.
But as my Polish is not very advanced, I wanted to doublecheck.

Reply:
Wyczyść ukończone is the literal translation of Clear completed. It is user-friendly and easy to understand. The translation depends on the context and what the option does in the UI.

Reply:
I don't have the option to build the application now. All I need is a screenshot (it doesn't matter what language you have the app in) to confirm that the translation is correct, if you want to be 100% sure.

Reply:
![image](https://github.com/Nexus-Mods/NexusMods.App/assets/6697380/f7672075-8a6f-4c69-b27f-e4005c322b0d)

Nevermind, it is 'clear completed'. Not a response to a click. just the static text.
So your translation is ok. My bad for not checking the UI before commenting.

Reply:
Static or not, in either case the translation is correct. Thanks for the screenshot.